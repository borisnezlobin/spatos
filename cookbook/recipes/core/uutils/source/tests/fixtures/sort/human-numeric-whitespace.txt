

456K

			456M


4568K

              6.2G

