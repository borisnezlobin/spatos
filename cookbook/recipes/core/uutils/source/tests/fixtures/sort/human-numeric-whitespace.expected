






456K
4568K
			456M
              6.2G
